hub/config/src/main/java/uk/gov/ida/hub/config/application/CertificateService.java

```diff
@@ -50,6 +55,12 @@ public  Certificate encryptionCertificateFor(String entityId) {
 
     public List<Certificate> signatureVerificationCertificatesFor(String entityId) {
         T config = getConfig(entityId);
+
+        config.getSignatureVerificationCertificates()
+                .stream()
+                .filter(cd -> !certificateValidityChecker.isValid(cd))
+                .forEach(cd -> LOG.warn("Signature verification certificates were requested for entityId '{}' but at least one is invalid", entityId));
+
```

Comment:
We could call `getSignatureVerificationCertificates().stream()` once then partition on `isValid()`. We could then handle the true case and the false case. Might be nicer?